### PR TITLE
feat: Setting up react-navigation for routes

### PR DESCRIPTION
**Pull request by NelsonPRSousa**

### CODE REVIEW

File: src/components/common/ScreenTitle.tsx [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import { Text } from 'react-native';
+
+type Props = {
+    children: string;
```
Comment (joaopedrodcf) [inline src/components/common/ScreenTitle.tsx]: Shouldn't this be a React.ReactNode cause we can put anything inside screen title and not just a string, but tests this out

File: src/index.tsx [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+import { StatusBar, Platform } from 'react-native';
+import { ThemeProvider } from 'styled-components';
+
+import ApplicationNavigator from './routes';
+import styles from './styles';
+
+import './config/ReactotronConfig';
+
+const App = (): Object => (
+    <Fragment>
```
[the remark this answers is not in the document]
Comment (NelsonPRSousa) [reply src/index.tsx]: Better, thanks for the tip! 👍

File: src/utils/isEqualsOrLargestThanIphoneX.ts [hunk 3]
```diff
@@ -0,0 +1,22 @@
+import { Dimensions, Platform } from 'react-native';
+
+const screenWidth = Dimensions.get('window').width;
+const screenHeight = Dimensions.get('window').height;
+
+const IPHONEX_WIDTH = 375;
+const IPHONEX_HEIGHT = 812;
+
+const isEqualsOrLargestThanIphoneX = (): boolean => {
```
[the remark this answers is not in the document]
Comment (NelsonPRSousa) [reply src/utils/isEqualsOrLargestThanIphoneX.ts]: Yes, of course 🦆

File: src/components/common/ScreenTitle.tsx (hunk 1 above)
Comment (NelsonPRSousa) [reply src/components/common/ScreenTitle.tsx]: Done, thanks for the tip 👍

**Pull request merged**